packages/client/src/client-details.ts

```diff
@@ -30,13 +35,26 @@ export const getDeviceInfo = () => {
   return deviceInfo;
 };
 
-export const getClientDetails = (): ClientDetails => {
+export const getWebRTCInfo = () => {
+  return webRtcInfo;
+};
+
+export const setWebRTCInfo = (info: WebRTCInfoType) => {
+  webRtcInfo = info;
+};
+
+export type LocalClientDetailsType = ClientDetails & {
+  webRTCInfo?: WebRTCInfoType;
+};
+
+export const getClientDetails = (): LocalClientDetailsType => {
```

Comment:
I wouldn't change the return type of this method as it is used in `Call.join()`.

I'm perfectly fine with using `getWebRtcVersion()` in `SfuStatsReporter`